docs/compression_algorithms/Quantization.md

```diff
@@ -268,8 +268,8 @@ sparsity and filter pruning algorithms. It can be enabled by setting a non-zero
             ]
         }
         "batchnorm_adaptation": {
-            "num_bn_adaptation_steps": 10, // Number of batches from the training dataset to pass through the model at initialization in order to update batchnorm statistics of the original model
-            "num_bn_forget_steps": 5, // Number of batches from the training dataset to pass through the model at initialization in order to erase batchnorm statistics of the original model (using large momentum value for rolling mean updates)
+            "num_bn_adaptation_samples": 2048, // Number of batches from the training dataset to pass through the model at initialization in order to update batchnorm statistics of the original model
+            "num_bn_forget_steps": 1024, // Number of batches from the training dataset to pass through the model at initialization in order to erase batchnorm statistics of the original model (using large momentum value for rolling mean updates)
```

Comment:
`"num_bn_forget_samples"`. Should also update the comments at corresponding lines to refer to samples instead of steps.